python3/cloudwatch/src/lambda_function.py

```diff
@@ -161,6 +178,12 @@ def lambda_handler(event, context):
     additional_data = _get_additional_logs_data(aws_logs_data, context)
     shipper = LogzioShipper()
 
+    whitelist_regexps.extend(_get_list_from_env_var("WHITELIST", "WHITELIST_SEPARATOR"))
+    blacklist_regexps.extend(_get_list_from_env_var("BLACKLIST", "BLACKLIST_SEPARATOR"))
```

Comment:
Since they're mutually exclusive, we only need 1 separator.
Also, please have WHITELIST, BLACKLIST, and LIST_SEPARATOR as constants in the beginning of the script, and use them.